CONTRIBUTING.md

```diff
@@ -28,7 +28,7 @@ $ gh pr create
 
 Build the manifests, install the CRD and run the operator as a local process:
 ```
-$ make manifests install run
+$ make bundle install run
```

Comment:
changing `manifests` to `bundle` because I see bundle internally runs `manifests` already, so running `bundle` also generates CRD level changes if any made.  

Also, If my understanding is right based on commit history. `make manifests` is added in docs prior to adding the `make bundle` target in Makefile. So now changing it to `make bundle` makes more sense for auto-generating the files.